.github/workflows/ci.yml

```diff
@@ -45,7 +45,30 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: check
-          args: --all-targets
+          args: --all-targets --all-features
```

Comment:
I guess it also would be reasonable to run cargo check on different feature combinations since it's pretty fast

Reply:
I might actually go back on this, since it doesn't always make sense to have all features enabled (some may be incompatible with others).